releases.md

```diff
@@ -45,49 +45,49 @@ As of 2019-03-29, PELUX 3.1 has been released! Documentation can be found
 
 |Sources for all the images| Commit hash |
 | ------------------------ | ----------- |
-| [Source release](https://pelux.io/artifacts/pelux/3.1/sources/source-release/) | 6266e8bcb8247d54b32119eefc72e93687b693be |
+| [Source release](https://pelux.io/artifacts/pelux/4.0/sources/source-release/) | 6266e8bcb8247d54b4.019eefc72e93687b693be |
```

Comment:
What is this commit hash for? It is not pelux-manifests. I can not find the hash. And hash should be changed?

Reply:
OK, it is pelux-manifests. I could not find the hash because there is a . there. But even without . hash is wrong.

A bit out of scope I guess but perhaps we should make it clearer that it is pelux-manifests we are referring to.